salt/docker/stop.sls

```diff
@@ -0,0 +1,4 @@
+# Stop and disable the docker daemon
+docker:
+  service.dead:
+    - enable: False
```

Comment:
does this use systemd? if not, if not we need to add containerd as well as part of the shutdown process, otherwise the node will hang because containerd exists before the running containers causing the node to just hang.